setup.py

```diff
@@ -166,5 +166,8 @@ def run(self):
     zip_safe=False,
     setup_requires=setup_requires,
     install_requires=install_requires,
-    use_scm_version=True,
+    use_scm_version={"local_scheme": "no-local-version",
```

Comment:
Should I use 'no-local-version' in MorphIO as well ?

Reply:
maybe - it's so I can upload to the test.pypi.org